test/old/testdir/Makefile

```diff
@@ -121,7 +121,8 @@ CLEAN_FILES := *.out \
   .*.swo            \
   .gdbinit          \
   $(TMPDIR)         \
-  del
+  del               \
+  $(ROOT)/runtime/doc/.index.txt.swp
```

Comment:
Why only this swp?

Reply:
Its the one that caused me trouble. Which other should I remove?

Reply:
```suggestion
  $(ROOT)/runtime/doc/.*.swp
```